src/steth/Swapper.sol

```diff
@@ -173,4 +226,42 @@ contract Swapper {
         // eth to weth
         weth.deposit{value: address(this).balance}();
     }
+
+    function swapWethToSdai(uint256 _wethAmount, uint256 _sDaiAmountOutMin) external returns (uint256 sDaiReceived) {
+        // weth => usdc => dai
+        uint256 daiAmount = uniswapSwapExactInputMultihop(
+            C.WETH, _wethAmount, 1, abi.encodePacked(C.WETH, uint24(500), C.USDC, uint24(100), C.DAI)
+        );
+
+        sDaiReceived = _swapDaiToSdai(daiAmount);
+
+        require(sDaiReceived > _sDaiAmountOutMin, "too little asset received");
```

Comment:
replace string with "AmountReceivedBelowMin" error and test all added functions in the Swapper.t.sol